src/Driver/MySQL/Schema/MySQLTable.php

```diff
@@ -79,6 +86,17 @@ protected function initSchema(State $state): void
         )->fetch()['Engine'];
     }
 
+    protected function isIndexColumnSortingSupported(): bool
```

Comment:
ok, though I'm not sure if we need to cache it, it should be pretty quick query

Reply:
It's not about caching it, but rather not calling it unless needed.

Reply:
Oh, you meant in the function. Would be odd to query for every index in the table.